doc/api/tls.md

```diff
@@ -1578,6 +1578,14 @@ secureSocket = tls.TLSSocket(socket, options);
 
 where `secureSocket` has the same API as `pair.cleartext`.
 
+### tls.roots
+<!-- YAML
+added: REPLACEME
+-->
+
+An immutable array of strings representing the root certificates (in PEM format)
+used for outbound TLS connections.
```

Comment:
would prefer: use for verifying peer certificates.

"outbound" is correct of course, but I think above aligns better with docs elsewhere.

Reply:
Or perhaps easiest, just say its the default value of the `ca` option, and point to https://nodejs.org/api/tls.html#tls_tls_createsecurecontext_options